conf/defaults.ini

```
@@ -255,6 +255,11 @@ min_refresh_interval = 5s
 # Path to the default home dashboard. If this value is empty, then Grafana uses StaticRootPath + "dashboards/home.json"
 default_home_dashboard_path =
 
+################################### Datasources #########################
```

As far as I recall, we're standardizing on writing it as "data sources".
```suggestion
################################### Data sources #########################
```